lib/msal-browser/test/interaction_client/PopupClient.spec.ts

```diff
@@ -1584,6 +1586,58 @@ describe("PopupClient", () => {
             });
         });
 
+        it("returns hash in query form when serverResponseType in OIDCOptions is query", (done) => {
```

Comment:
```suggestion
        it("returns server code response in query form when serverResponseType in OIDCOptions is query", (done) => {
```

In this context, `hash` used to refer to the server auth code response because we only supported "hash" mode, so the thing we were parsing was always a "hash". Now that we support `query` in OIDC as well, calling it a "hash" is no longer accurate.